src/templates/index.pug

```diff
@@ -1,5 +1,11 @@
 include _common.pug
 +head(title="C++ Tour")
+-
+    const links = [
+        { title: "Twitter", href: "#1", img: require('../img/icons/twitter.svg')},
+        { title: "Github", href: "#1", img: require('../img/icons/github.svg')},
```

Comment:
I assume at least github link should point to our repository :)

Reply:
I can change the links later, no problem.